node/src/reactor/main_reactor/validate.rs

```diff
@@ -38,8 +44,23 @@ impl MainReactor {
 
         match self.storage.read_highest_complete_block() {
             Ok(Some(highest_complete_block)) => {
+                // If we're lagging behind the rest of the network, fall back out of Validate mode.
+                let sync_identifier = SyncIdentifier::LocalTip(
+                    *highest_complete_block.hash(),
+                    highest_complete_block.height(),
+                    highest_complete_block.header().era_id(),
+                );
+                match self.block_accumulator.sync_instruction(sync_identifier) {
```

Comment:
Just to be sure - for a block too old this will create the `LeapInstruction::OutsideAttemptExecutionThreshold`, which - in turn, will create `SyncInstruction::Leap`, right?

Reply:
Yes, I believe so.